llm/llama-4-finetuning/README.md

```diff
@@ -75,23 +83,38 @@ run: |
   --rdzv_endpoint=$MASTER_ADDR:29500 \
   full_finetune_distributed \
   --config llama4/maverick_17B_128E_full \
-  model_dir=/tmp/Llama-4-Maverick-17B-128E-Instruct
+  model_dir=/tmp/Llama-4-Maverick-17B-128E-Instruct \
+  dataset.packed=True tokenizer.max_seq_len=4096 \
+  gradient_accumulation_steps=1 \
+  enable_activation_offloading=True \
+  activation_offloading_use_streams=False \
+  optimizer_in_bwd=True \
+  optimizer=torch.optim.AdamW \
+  optimizer_kwargs.fused=True \
+  max_steps_per_epoch=1 \
+  epochs=10 \
+  enable_dcp=True \
+  enable_async_checkpointing=True \
+  resume_from_checkpoint=False \
+  keep_last_n_checkpoints=1 \
+  fsdp_cpu_offload=True
 ```
     
 </details>
 
-Run the following on your local machine:
-
 ```bash
-# Download the files for Llama 4 finetuning
-git clone https://github.com/skypilot-org/skypilot
-cd skypilot/llm/llama-4-finetuning
+sky launch -c maverick-torchtune llama-4-maverick-sft.yaml \
+  --env HF_TOKEN
+```
 
-export HF_TOKEN=xxxx
+## Alternative Approaches
 
-# Full finetuning of Llama 4 Maverick 17B MoE
-# Requires 2+ nodes with H200 GPUs for distributed training
-sky launch -c maverick llama-4-maverick.yaml \
+### LoRA Fine-tuning (Lower Resource Requirements)
+For users with limited GPU resources, LoRA (Low-Rank Adaptation) provides an efficient alternative:
+
+```bash
+# LoRA finetuning - requires fewer resources
+sky launch -c maverick-lora llama-4-maverick-llama-factory-lora.yaml \
```

Comment:
YAML path probably needs to be updated?

Reply:
Updated all references to point to correct yaml